src/Models/Scopes/WithMirasvitLabelsScope.php

```diff
@@ -19,7 +19,10 @@ public function apply(Builder $builder, Model $model)
                 "cat_position", cat_position.code,
                 "cat_style", cat.style
         )), "$.null__") as mirasvit_label')
-            ->leftJoin('mst_productlabel_index as label_index', 'label_index.product_id', '=', $model->getTable() .'.entity_id')
+            ->leftJoin('mst_productlabel_index as label_index', function($join) use ($model) {
+                $join->on('label_index.product_id', '=', $model->getTable() .'.entity_id')
+                    ->where('label_index.store_id', config('rapidez.store'));
```

Comment:
Does the index always get linked to the specific store, no `store_id` = 0?
If that is the case, this current implementation will miss those labels

Reply:
Yes, if no specific store view is entered then this is put down in the index table for all existing store views.